eslint.config.js

```diff
@@ -0,0 +1,17 @@
+import pluginVue from "eslint-plugin-vue";
+import eox from "@eox/eslint-config";
+
+export default [
+  ...eox,
+  ...pluginVue.configs["flat/essential"],
+  {
+    ignores: ["public/", "dist/", ".eodash/"],
+  },
+  {
+    files: ["core/client/**/*.vue", "widgets/**/*.vue"],
+    rules: {
+      "vue/no-deprecated-html-element-is": "warn",
+      "vue/multi-word-component-names": "off",
```

Comment:
and i imagine some multi word component names, this should be fine i think

Reply:
I thought we don't have that much Vue components in the project but I don't mind renaming them to multi words generally

Reply:
ah, so the problem is that they are not multiword.... i don't have a strong opinion on this, if you think this makes sense we can keep it disabled